docs/layouts/shortcodes/generated/pulsar_source_configuration.html

```diff
@@ -0,0 +1,54 @@
+        </tr>
+        <tr>
+            <td><h5>pulsar.source.maxFetchRecords</h5></td>
+            <td style="word-wrap: break-word;">100</td>
+            <td>Integer</td>
+            <td>The max number of records to fetch to wait when polling. A longer time increases throughput but also latency. A fetch batch might be finished earlier because of <code class="highlighter-rouge">pulsar.source.maxFetchTime</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.source.maxFetchTime</h5></td>
+            <td style="word-wrap: break-word;">10000</td>
+            <td>Long</td>
+            <td>The max time (in ms) to wait when fetching records. A longer time increases throughput but also latency. A fetch batch might be finished earlier because of <code class="highlighter-rouge">pulsar.source.maxFetchRecords</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.source.partitionDiscoveryIntervalMs</h5></td>
+            <td style="word-wrap: break-word;">30000</td>
+            <td>Long</td>
+            <td>The interval (in ms) for the Pulsar source to discover the new partitions. A non-positive value disables the partition discovery.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.source.transactionTimeoutMillis</h5></td>
+            <td style="word-wrap: break-word;">10800000</td>
+            <td>Long</td>
+            <td>This option is used for when using <code class="highlighter-rouge">Shared</code> or <code class="highlighter-rouge">Key_Shared</code> subscription. You should config this option when you didn't enable the <code class="highlighter-rouge">pulsar.source.enableAutoAcknowledgeMessage</code> option.<br />The value (in ms) should be greater than the checkpoint interval.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.source.verifyInitialOffsets</h5></td>
+            <td style="word-wrap: break-word;">WARN_ON_MISMATCH</td>
+            <td><p>Enum</p></td>
+            <td>Upon (re)starting the source checks whether the expected message can be read. If failure is enabled the application fails, else it logs a warning. A possible solution is to adjust the retention settings in pulsar or ignoring the check result.<br /><br />Possible values:<ul><li>"FAIL_ON_MISMATCH"</li><li>"WARN_ON_MISMATCH"</li></ul></td>
```

Comment:
```suggestion
            <td>Upon (re)starting the source, check whether the expected message can be read. If failure is enabled, the application fails. Otherwise, it logs a warning. A possible solution is to adjust the retention settings in Pulsar or ignoring the check result.<br /><br />Possible values:<ul><li>"FAIL_ON_MISMATCH"</li><li>"WARN_ON_MISMATCH"</li></ul></td>
```